### ISSUE TITLE
Heartbeats

### ISSUE THREAD

**Issue by moshloop** [flanksource/canary-checker#1542]
```yaml
spec:
    schedule: @every 1h
    heartbeat: "2h"
```

Create a job that will insert a check status of false and error = true, with "Heartbeat failed" as the message

+ last_result function should ignore all error = true check statuses
